test/utils/snap.go

```diff
@@ -83,9 +83,10 @@ func SnapDumpLogs(t *testing.T, name string) {
 
 func SnapLogsJournal(t *testing.T, start time.Time, name string) string {
 	logs, _ := Exec(t, fmt.Sprintf(
-		"sudo journalctl --since \"%s\" --no-pager | grep \"%s\"\n\n",
-		time.Now().Format("2006-01-02 15:04:05"),
+		"sudo journalctl --since \"%s\" --no-pager | grep \"%s\"|| true\n\n",
+		start.Format("2006-01-02 15:04:05"),
```

Comment:
👀

Reply:
Oh... Do I actually need a for loop

Reply:
Ok, it works. I removed the loop.

Reply:
Yes this was a bug. 

Maybe on a slow system, 1 second would not be sufficient, but we can address that if it happens.